systems/identity/identity-application/src/main/kotlin/hs/kr/entrydsm/identity/application/service/PassService.kt

```
@@ -0,0 +1,120 @@
+            throw exception.toIdentityException()
+        }
+        if (identity.phoneNumber.isBlank() || identity.name.isBlank()) {
+            throw IdentityDomainException(ErrorCode.INVALID_PASS)
+        }
+        try {
+            if (!callbackTokenStore.claim(token, proofTtlSeconds)) {
+                throw IdentityDomainException(ErrorCode.INVALID_PASS)
+            }
+        } catch (exception: IdentityDomainException) {
+            throw exception
+        } catch (exception: RuntimeException) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)